README.md

```diff
@@ -201,6 +202,18 @@ Except C++ and Python API oneDAL also provide API for C++ SYCL and Java language
 - [Release Notes](https://software.intel.com/content/www/us/en/develop/articles/oneapi-dal-release-notes.html)
 - [Known Issues](https://oneapi-src.github.io/oneDAL/notes/known_issues.html)
 
+## Support
+Report issues, ask questions, and provide suggestions using:
```

Comment:
Makes sense to add Discussions as well. I don't like the idea that your propose to use github issues to ask questions, because issues workflow is not suitable for that.

Reply:
We added discussions later.